docs/tutorial/maps.rst

```diff
@@ -181,748 +168,387 @@ Some basic statistical functions are built into Map objects:
+This makes it simple to plot *any* coordinate on top of our Map using the :meth:`~astropy.visualization.wcsaxes.WCSAxes.plot_coord` method.
+The following example shows how to plot some points on our Map, including the center coordinate of our Map.
 
-This clips out the dimmest 1% of pixels and the brightest 0.5% of pixels.
-With those outlier pixels clipped, the resulting image makes better use of the full range of colors.
-If you'd like to see what areas of your images got clipped, you can modify the colormap:
+.. plot::
+    :include-source:
+    :context: close-figs
 
-.. code-block:: python
+    coords = SkyCoord(Tx=[100,1000] * u.arcsec, Ty=[100,1000] * u.arcsec, frame=my_map.coordinate_frame)
+    fig = plt.figure()
+    ax = fig.add_subplot(projection=my_map)
+    my_map.plot(axes=ax, clip_interval=(1,99.5)*u.percent)
+    ax.plot_coord(coords, 'o')
+    ax.plot_coord(my_map.center, 'X')
+    plt.show()
 
-    >>> cmap = map.cmap  # doctest: +SKIP
-    >>> cmap.set_over('blue')  # doctest: +SKIP
-    >>> cmap.set_under('green')  # doctest: +SKIP
+.. note::
 
-This will color the areas above and below in red and green respectively (similar to this `matplotlib example <https://matplotlib.org/examples/pylab_examples/image_masked.html>`__).
-You can use the following colorbar command to display these choices:
+    Map visualizations can be heavily customized using both `matplotlib` and `astropy.visualization.wcsaxes`.
+    See the :ref:`Plotting section of the Example Gallery <sphx_glr_generated_gallery_plotting>` for more detailed examples of how to customize Map visualizations.
 
-.. code-block:: python
+.. _cropping-maps:
 
-    >>> plt.colorbar(extend='both')   # doctest: +SKIP
+Cropping Maps and Combining Pixels
+==================================
 
-Here is an example of this put to use on an AIA image.
+It analyzing images of the Sun, we often want to choose a smaller portion of the full disk to look at more closely.
+Let's use the region of interest we defined above to crop out that portion of our image.
 
 .. plot::
     :include-source:
+    :context: close-figs
 
-    import astropy.units as u
-    import matplotlib.pyplot as plt
-
-    import sunpy.map
-    import sunpy.data.sample
-
-    smap = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
-    cmap = smap.cmap.copy()
-    cmap.set_over('blue')
-    cmap.set_under('green')
-
-    fig = plt.figure(figsize=(12, 4))
+    my_submap = my_map.submap(roi_bottom_left, top_right=roi_top_right)
+    fig = plt.figure()
+    ax = fig.add_subplot(projection=my_submap)
+    my_submap.plot(axes=ax)
+    plt.show()
 
-    ax1 = fig.add_subplot(1, 2, 1, projection=smap)
-    smap.plot(title='Without clipping')
-    plt.colorbar()
+Additionally, we also may want to combine multiple pixels into a single pixel (a so called "superpixel") to, for example, increase our signal-to-noise ratio.
```

Comment:
is the word "combine" here confusing? should it be resample? or something else?

Reply:
I think combine here is the right word. The image is not being resampled, its being rebinned such that multiple pixels are actually being combined into a single new pixel.